reana_workflow_engine_serial/tasks.py

```diff
@@ -10,14 +10,15 @@
 
 from __future__ import absolute_import, print_function
 
-from distutils.dir_util import copy_tree
 import json
 import logging
 import os
+from distutils.dir_util import copy_tree
 from time import sleep
 
 from reana_commons.api_client import JobControllerAPIClient
 from reana_commons.publisher import WorkflowStatusPublisher
+from reana_commons.tasks import stop_workflow
```

Comment:
This approach has a big limitation since we are running long living tasks, it is very likely to happen that there is no free process to run the `stop_workflow` task, so stop won't be working when engines are fully busy. We can solve this by moving this task to a `reana-worker`, following the work started by @dinosk in https://github.com/reanahub/reana-cluster/pull/91. cc @reanahub/developers